test/common/grpc/google_grpc_utils_test.cc

```diff
@@ -0,0 +1,81 @@
+#include <arpa/inet.h>
+
+#include "common/grpc/google_grpc_utils.h"
+
+#include "test/mocks/upstream/mocks.h"
+#include "test/proto/helloworld.pb.h"
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Grpc {
```

Comment:
Nit: tests should be further nested in an anonymous namespace.

Reply:
done